WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -32,34 +52,28 @@ class GutenbergLayoutPickerViewController: UIViewController {
         }
     }
 
-    var completion: PageCoordinator.TemplateSelectionCompletion? = nil
```

Comment:
This was just moved lower in the section of vars largely just so I would still see it.